src/raftstore/store/store.rs

```diff
@@ -1691,17 +1691,14 @@ impl<T: Transport, C: PdClient> Store<T, C> {
     }
 
     fn handle_snap_mgr_gc(&mut self) -> Result<()> {
-        let mut snap_keys = try!(self.snap_mgr.list_snap());
+        let mut snap_keys = try!(self.snap_mgr.list_idle_snap());
```

Comment:
The snapshot will not be deleted. Check this

https://github.com/pingcap/tikv/blob/hhkbp2/refactor-snapshot-manager/src/raftstore/store/snap.rs#L2065